python/cudf/cudf/tests/test_parquet.py

```diff
@@ -626,6 +626,7 @@ def test_parquet_reader_spark_timestamps(datadir):
     assert_eq(expect, got)
 
 
+@pytest.mark.xfail(reason="decimal128 not yet supported in cuDF")
```

Comment:
should we modify this test to expect a failure instead of adding a new file?

Reply:
Updated the tests accordingly 👍